deploy/operator/ztp/assisted-installer-crds-playbook.yaml

```diff
@@ -5,6 +5,9 @@
   gather_facts: no
   vars:
     - spoke_namespace: "{{ lookup('env', 'SPOKE_NAMESPACE') }}"
+    - spoke_controlplane_agents: "{{ lookup('env', 'SPOKE_CONTROLPLANE_AGENTS') }}"
+    - spoke_api_vip: "{{ lookup('env', 'SPOKE_API_VIP') }}"
+    - spoke_ingress_vip: "{{ lookup('env', 'SPOKE_INGRESS_VIP') }}"
```

Comment:
We could move these to command line arguments instead of environment...outside the scope of this PR but it's something to consider.